distributed/comm/tcp.py

```diff
@@ -240,11 +240,12 @@ async def read(self, deserializers=None):
             self._closed = True
             if not sys.is_finalizing():
                 convert_stream_closed_error(self, e)
-        except Exception:
-            # Some OSError or a another "low-level" exception. We do not really know what
-            # was already read from the underlying socket, so it is not even safe to retry
-            # here using the same stream. The only safe thing to do is to abort.
-            # (See also GitHub #4133).
+        except BaseException:
```

Comment:
when #4133 was introduced CancelledError was still a subclass of Exception since 3.8 CancelledError is no-longer caught here